Update obsolete Scoped Custom Element Registry tests

annevk opened the pull request:
Update tests that don't appear to duplicate the "Revamped" tests to
match the new API surface.

The remainder are removed.

Historical tests are added at the same time for features we decided not
to standardize. Also remove some historical tests that have coverage
elsewhere.

domenic commented on custom-elements/historical.html:
```diff
@@ -1,35 +1,32 @@
-    assert_false(element.hasAttribute('is'))
-  } catch (e) {
-    // If v1 is supported, then converting string to dictionary should throw.
-    assert_throws_js(TypeError, function() { throw e })
```
This change seems like it changes the meaning of the test, which requires that `createElement('x', 'string')` throws.

annevk replied:
That was a bogus claim as we have ignored string arguments in the DOM standard since essentially the introduction of this feature.

annevk commented:
I was under the mistaken impression that we had attempted to reuse all these tests already, but that does not appear to be the case. I now think more should be adapted to the newer API syntax.

annevk committed:
Update obsolete Scoped Custom Element Registry tests

Update tests that don't appear to duplicate the "Revamped" tests to
match the new API surface.

The remainder are removed.

Historical tests are added at the same time for features we decided not
to standardize. Also remove some historical tests that have coverage
elsewhere.

annevk committed:
Update scoped-registry-define-upgrade-criteria.tentative.html

rniwa commented on custom-elements/revamped-scoped-registry/scoped-registry-define-upgrade-criteria.tentative.html:
```diff
@@ -245,14 +245,11 @@
-
 </script>
-
-</body>
```
This seems like unrelated change?

rniwa commented on custom-elements/revamped-scoped-registry/scoped-registry-define-upgrade-order.tentative.html:
```diff
@@ -214,4 +214,3 @@
 }, 'Elements in the "owner" window of a scoped registry are not always upgraded first');
 
 </script>
-</body>
```
Ditto.

rniwa commented on custom-elements/revamped-scoped-registry/scoped-registry-registry-define-get-etc.tentative.html:
```diff
@@ -101,4 +101,3 @@
 }, 'Scoped Custom element registry getters do not resolve globally registered classes from whenDefined');
 
 </script>
-</body>
```
Ditto.
